application/features/user/user.feature

```diff
@@ -0,0 +1,196 @@
+Feature: User
+  In order to identify users
+  As an authorized user
+  I need to be able to manage user information
+
+  Scenario: Attempt to create a user as an unauthorized user
+    Given the requester is not authorized
+      And there are no users yet
+    When I request the user be created
+    Then the response status code should be 401
+      And the property message should contain "invalid credentials"
+      And a user was not created
+
+  Scenario Outline: Attempt to act upon a user in an undefined way
+    Given the requester is authorized
+      And there are no users yet
+    When I request the user be <action>
+    Then the response status code should be 404
+      And the property message should contain "not found"
+      And a user was not created
+
+    Examples:
+      | action    |
+      | retrieved |
+      | updated   |
+      | deleted   |
+
+  Scenario Outline: Attempt to create a new user without providing a required property
```

Comment:
glad that was helpful, I know it certainly saved me some time once I got the hang of it!